ReactWindows/ReactNative.Shared/UIManager/RootViewHelper.cs

```diff
@@ -99,12 +100,22 @@ private static DependencyObject GetParent(DependencyObject view, bool findRoot)
 
             if (!findRoot)
             {
-                return (view as FrameworkElement)?.Parent;
-            }
-            else
-            {
-                return VisualTreeHelper.GetParent(view);
+                var frameworkElement = view as FrameworkElement;
+                if (frameworkElement != null)
+                {
+                    return frameworkElement.Parent;
+                }
+#if !WINDOWS_UWP
+
```

Comment:
maybe rename these two variables to just "element" with #if UWP for the different casts, and then deduplicate the if+return?

Reply:
I think you're reading it wrong, we had to add the extra check for WPF because there is a difference between RichTextBlock and TextBlock in WPF.

---
In reply to: [132557372](https://github.com/Microsoft/react-native-windows/pull/1256#discussion_r132557372) [](ancestors = 132557372)